Services/Sessions/WebAppSession.m

```diff
@@ -299,4 +299,28 @@ - (ServiceSubscription *)subscribeMediaInfoWithSuccess:(SuccessBlock)success fai
     return nil;
 }
 
+#pragma mark - Playlist Control
+
+- (id <PlayListControl>)playListControl
+{
+    return nil;
+}
+
+- (CapabilityPriorityLevel)playListControlPriority
+{
+    return CapabilityPriorityLevelVeryLow;
```

Comment:
Why don't we change them to something like "NOT_SUPPORTED"? Android team added a new CapabilityPriorityLevel for some reason before. https://github.com/ConnectSDK/Connect-SDK-Android-Core/blob/dev/src/com/connectsdk/service/capability/CapabilityMethods.java#L39